src/transport_performance/gtfs/validation.py

```diff
@@ -242,9 +242,74 @@ def clean_feed(self):
             # TODO: Issue 74 - Improve this to clean feed when KeyError raised
             print("KeyError. Feed was not cleaned.")
 
+    def _produce_stops_map(
```

Comment:
Hi @CBROWN-ONS - this likely deserves more of a conversation so that I can understand the scenario. But as it's not part of the diff in this PR, I'll put it in an issue with a 'needs triage' label.